portfolio/src/main/webapp/script.js

```diff
@@ -21,12 +21,12 @@ function getRandomGreeting() {
   fetch('/data').then(response => response.json()).then((greeting) => {
     const listElement = document.getElementById('greeting-container');
     listElement.innerHTML = '';
-    listElement.appendChild (
-      createListElement(greeting[0] + ' '));
-    listElement.appendChild (
-      createListElement(greeting[1] + ' '));
-    listElement.appendChild (
-      createListElement(greeting[2] + ' '));
+
+    var i;
+    for(i=0; i<greeting.length; i++) {
```

Comment:
Because i is only used in the scope of the loop, it's better practice to keep it inline:

for (var i=0, i < greeting.length; i++) {..}

Reply:
Done!